File: Modul-1/Soal 4/PRAK104-2310817110013-RandyFebrian.py
a, b, diskonA, diskonB = 400000, 350000, 13, 21
hasilA = round(a - (a * diskonA / 100))
hasilB = round(b - (b * diskonB / 100))

print("Harga sepatu A adalah", a)
print("Harga sepatu B adalah", b)
print("Sepatu A mendapat diskon {}% sehingga harganya menjadi {}".format(diskonA, hasilA))
print("Sepatu B mendapat diskon {}% sehingga harganya menjadi {}".format(diskonB, hasilB))

File: Modul-1/Soal 4/PRAK104-2310817110013-RandyFebrian.c
#include <stdio.h>

int main(){
    int sepatuA = 400000, sepatuB = 350000;
    int diskon1 = 13, diskon2 = 21;
    int hasilA = sepatuA - (sepatuA * diskon1 /100), hasilB = sepatuB - (sepatuB * diskon2 /100);

    printf("Harga sepatu A adalah %d\n", sepatuA);
    printf("Harga sepatu B adalah %d\n", sepatuB);
    printf("Sepatu A mendapat diskon %d%% sehingga harganya menjadi %d\n", diskon1, hasilA );
    printf("Sepatu B mendapat diskon %d%% sehingga harganya menjadi %d\n", diskon2, hasilB );  
    return 0;  
}